Sources/Basics/Archiver/ZipArchiver.swift

```diff
@@ -48,7 +70,9 @@ public struct ZipArchiver: Archiver, Cancellable {
             }
 
             #if os(Windows)
-            let process = AsyncProcess(arguments: ["tar.exe", "xf", archivePath.pathString, "-C", destinationPath.pathString])
+            // FileManager lost the ability to detect tar.exe as executable.
```

Comment:
I wonder if that's related to https://github.com/swiftlang/swift-foundation/issues/860